### DISCUSSION TITLE
Haiku is a X-line poem with Y syllables, written in N-P-N syllable count ?

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#60] [Galaxy Qs] [answered]
What are the values of `X`, `Y`, `N` & `P`

**Comment by opticaldrive** [comment]
X = 3
y = 17 
N = 5
P = 7

**Comment by zoelesv** [comment, marked as answer]
X = 3
Y = 17 
N = 5 
P = 7 

Haiku is a 3-line poem with a syllable pattern of 5-7-5.

**Reply by Sriansh-raj** [reply]
Heya @zoelesv Welcome to `Galaxy Qs` 🐱 
Congratulations 🎉 You just got a `Galaxy Brain` achievement.

**Reply by Sriansh-raj** [reply]
Don't forget to create a PR and update your name in [Users_who_got_Galaxy_Brain.md](https://github.com/Sriansh-raj/GitHub-Achievements/blob/main/Users_who_got_Galaxy_Brain.md) to create a history 😃

**Reply by zoelesv** [reply]
Thank you! This is awesome.
